### DISCUSSION TITLE
Informazione di servizio per Marco: cambio layout del repo GIT

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#31] [General]
Credo sia interessato solo @MarcoJul: sto facendo qualche modifica al repo GIT in termini di branch:
 * il branch `staging` verrà cancellato poiché non ha più senso parlare di `main` e `staging` in questa fase visto che la versione on line è fissata da tempo e ha la tag `v8`, quindi non ci perdiamo niente e sappiamo cosa c'è on line;
 * fino a nuovi cambiamenti faccio merge del tuo lavoro dentro `main`: tu fai altrettanto. Insomma: `git merge paolo/main` o come cavolo hai chiamato il mio _remote_ sul tuo PC;
 * comparirà un branch `PG` in cui sperimento con PostgreSQL e _full text search_, puoi ignorare tranquillamente questo nuovo branch.

**Comment by pdonadeo** [comment]
Ho appena fatto commit di un paio di cose, compreso un fix sul frontend (un CSS "rotto" per React).

Quando puoi fai merge.

**Reply by MarcoJul** [reply]
Ottimo! Mi riallineo subito e continuo con il lavoro!

**Reply by MarcoJul** [reply]
Ciao Paolo, ho fatto merge dal tuo main (adesso dovremmo essere allineati come commit e tutto il resto) ma adesso ho questo errore che non mi riesco a spiegare🫣:

<img width="565" alt="image" src="https://user-images.githubusercontent.com/82112168/177644566-48d350ca-f9fa-4e33-917f-4f261b9d70f2.png">

Tu ne capisci qualcosa di più? Nel frattempo cerco ancora di analizzarlo..🤔

**Reply by pdonadeo** [reply]
Sto andando a dormire, domani ci guardo.

**Reply by MarcoJul** [reply]
Ritiro tutto, fesso io che ho incasinato le porte (avevo occupato la 3000 da altro senza accorgermene.. tutto ok!

**Comment by pdonadeo** [comment]
È tutto on line su `staging`.
